docker/airflow/entrypoint.sh

```diff
@@ -34,6 +34,21 @@ function header() {
 if [ "$1" == help ] || [ "$1" == --help ]; then help_text && exit 0; fi
```

Comment:
I can't comment on the exact line but for L29 and L31 - those two lines weren't behaving as expected after the environment modification steps 😞:

<details> <summary>Output</summary>

```
###########################################################################################################################################################################################################################################################################################
                                                                                                                                   MODIFYING ENVIRONMENT
###########################################################################################################################################################################################################################################################################################
Variable Name : AIRFLOW_CONN_SLACK_ALERTS
Original Value: https://hooks.slack.com/services/url
New Value:      https://https%3A%2F%2Fhooks.slack.com%2Fservices%2Furl
Variable Name : AIRFLOW_CONN_DATA_REFRESH
Original Value: http://172.17.0.1:8001
New Value:      https://http%3A%2F%2F172.17.0.1%3A8001
Variable Name : AIRFLOW_CONN_SLACK_NOTIFICATIONS
Original Value: https://hooks.slack.com/services/url
New Value:      https://https%3A%2F%2Fhooks.slack.com%2Fservices%2Furl
AIRFLOW_CONN_AWS_DEFAULT=aws://test_key:test_secret@?region_name=us-east-1&host=http://s3:5000
AIRFLOW_CONN_DATA_REFRESH=https://http%3A%2F%2F172.17.0.1%3A8001
AIRFLOW_CONN_EMR_EMPTY=emr://
AIRFLOW_CONN_EMR_TEST=emr://?host=http://s3:5000
AIRFLOW_CONN_POSTGRES_OPENLEDGER_TESTING=postgres://deploy:deploy@postgres:5432/openledger
AIRFLOW_CONN_POSTGRES_OPENLEDGER_UPSTREAM=postgres://deploy:deploy@postgres:5432/openledger
AIRFLOW_CONN_SLACK_ALERTS=https://https%3A%2F%2Fhooks.slack.com%2Fservices%2Furl
AIRFLOW_CONN_SLACK_NOTIFICATIONS=https://https%3A%2F%2Fhooks.slack.com%2Fservices%2Furl
#
                                                                                                                                   WAITING FOR POSTGRES
#
2022-04-14 00:10:11,209 - /opt/airflow/wait_for_db.py - INFO:  Testing DB connection postgresql+psycopg2://airflow:airflow@postgres:5432/airflow
#
                                                                                                                                    MIGRATING DATABASE
#
DB: postgresql+psycopg2://airflow:***@postgres:5432/airflow
/usr/local/airflow/.local/lib/python3.9/site-packages/sqlalchemy/dialects/postgresql/base.py:3571 SAWarning: Predicate of partial index idx_dag_run_queued_dags ignored during reflection
/usr/local/airflow/.local/lib/python3.9/site-packages/sqlalchemy/dialects/postgresql/base.py:3571 SAWarning: Predicate of partial index idx_dag_run_running_dags ignored during reflection
[2022-04-14 00:10:13,157] {db.py:919} INFO - Creating tables
INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
INFO  [alembic.runtime.migration] Will assume transactional DDL.
Upgrades done
#
                                                                                                                                      RUNNING "bash"
#
airflow@ccea441e1e76:~$ exit
exit
```

</details>

Do you think we could change those lines to Python for better consistency?
```bash
  python -c "print('#' * ${COLUMNS})"
  printf "%*s\n" $(( (${#1} + size) / 2)) "$1"
  python -c "print('#' * ${COLUMNS})"
```

Reply:
I think that changing IFS was causing this problem, and wasn't really necessary. This is what I see in the logs now, does that look better to you?

<details><summary>Output</summary>
```
################################################################################
                             MODIFYING ENVIRONMENT
################################################################################
Variable Name: AIRFLOW_CONN_SLACK_ALERTS
    Old Value: https://slack
    New Value: https://https%3A%2F%2Fslack
Variable Name: AIRFLOW_CONN_DATA_REFRESH
    Old Value: https://host.docker.internal:8001
    New Value: https://https%3A%2F%2Fhost.docker.internal%3A8001
Variable Name: AIRFLOW_CONN_SLACK_NOTIFICATIONS
    Old Value: https://slack
    New Value: https://https%3A%2F%2Fslack
Variable Name: AIRFLOW_CONN_EMR_TEST
    Old Value: emr://?host=http://s3:5000
    New Value: emr://?host=https://https%3A%2F%2Fs3%3A5000
Variable Name: AIRFLOW_CONN_AWS_DEFAULT
    Old Value: aws://test_key:test_secret@?region_name=us-east-1&host=http://s3:5000
    New Value: aws://test_key:test_secret@?region_name=us-east-1&host=https://https%3A%2F%2Fs3%3A5000
################################################################################
                              WAITING FOR POSTGRES
################################################################################
2022-04-19 23:24:01,406 - /opt/airflow/wait_for_db.py - INFO:  Testing DB connection postgresql+psycopg2://airflow:airflow@postgres:5432/airflow
################################################################################
                               MIGRATING DATABASE
################################################################################
DB: postgresql+psycopg2://airflow:***@postgres:5432/airflow
/usr/local/airflow/.local/lib/python3.9/site-packages/sqlalchemy/dialects/postgresql/base.py:3571 SAWarning: Predicate of partial index idx_dag_run_queued_dags ignored during reflection
/usr/local/airflow/.local/lib/python3.9/site-packages/sqlalchemy/dialects/postgresql/base.py:3571 SAWarning: Predicate of partial index idx_dag_run_running_dags ignored during reflection
[2022-04-19 23:24:02,376] {db.py:919} INFO - Creating tables
INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
INFO  [alembic.runtime.migration] Will assume transactional DDL.
Upgrades done
################################################################################
                              CREATING ADMIN USER
################################################################################
```</details>

Reply:
Dang, incredible find! Yea, this works great for me now 😄